aws-databrew-project/aws-databrew-project.json

```
@@ -0,0 +1,130 @@
+{
+  "typeName": "AWS::DataBrew::Project",
+  "description": "Resource schema for AWS::DataBrew::Project.",
+  "sourceUrl": "https://github.com/aws-cloudformation/aws-cloudformation-resource-providers-databrew.git",
+  "properties": {
+    "DatasetName": {
+      "description": "Dataset name",
+      "type": "string",
+      "minLength": 1,
+      "maxLength": 255
+    },
+    "Name": {
+      "description": "Project name",
+      "type": "string",
+      "minLength": 1,
+      "maxLength": 255
+    },
+    "RecipeName": {
+      "description": "Recipe name",
+      "type": "string",
+      "minLength": 1,
+      "maxLength": 255
+    },
+    "RoleArn": {
+      "description": "Role arn",
+      "type": "string"
+    },
+    "Sample": {
+      "type": "object",
+      "properties": {
+        "Size": {
+          "description": "Sample size",
+          "minimum": 1,
+          "type": "integer"
+        },
+        "Type": {
+          "description": "Sample type",
+          "enum": [
+            "FIRST_N",
+            "LAST_N",
+            "RANDOM"
+          ],
+          "type": "string"
+        }
+      },
+      "additionalProperties": false,
+      "required": [
+        "Type"
+      ]
+    },
+    "Tags": {
+      "type": "array",
+      "uniqueItems": false,
+      "items": {
+        "$ref": "#/definitions/Tag"
+      }
+    }
+  },
+  "definitions": {
+    "Tag": {
+      "description": "A key-value pair to associate with a resource.",
+      "type": "object",
+      "properties": {
+        "Key": {
+          "type": "string",
+          "minLength": 1,
+          "maxLength": 128
+        },
+        "Value": {
+          "type": "string",
+          "minLength": 0,
+          "maxLength": 256
+        }
+      },
+      "additionalProperties": false,
+      "required": [
+        "Value",
+        "Key"
+      ]
+    }
+  },
+  "additionalProperties": false,
+  "required": [
+    "DatasetName",
+    "Name",
+    "RecipeName",
+    "RoleArn"
+  ],
+  "primaryIdentifier": [
+    "/properties/Name"
+  ],
+  "createOnlyProperties": [
+    "/properties/Name",
+    "/properties/Tags"
+  ],
+  "handlers": {
+    "create": {
+      "permissions": [
+        "databrew:CreateProject",
+        "databrew:TagResource",
+        "databrew:UntagResource",
+        "iam:PassRole"
+      ]
+    },
+    "read": {
+      "permissions": [
+        "databrew:ReadProject",
```

```suggestion
        "databrew:DescribeProject",
```
https://docs.aws.amazon.com/service-authorization/latest/reference/list_awsgluedatabrew.html